### DISCUSSION TITLE
Can't Install It

### DISCUSSION THREAD

**Discussion by darshanpjpt** [projectdiscovery/naabu#713] [General]
## Error
```
github.com/google/gopacket/pcap
# github.com/google/gopacket/pcap
../../go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap_unix.go:34:10: fatal error: pcap.h: No such file or directory
   34 | #include <pcap.h>
      |          ^~~~~~~~
compilation terminated.
```

**Comment by MetzinAround** [comment]
I'm not sure what you're trying to install here, it doesn't look like naabu

**Comment by MetzinAround** [comment]
To install, follow installation instructions [here](https://github.com/projectdiscovery/naabu#installation-instructions) or use [PDTM](https://github.com/projectdiscovery/pdtm)

**Comment by darshanpjpt** [comment]
This is error while running `go install`

You sure It doesn't look like naabu?

![image](https://github.com/projectdiscovery/naabu/assets/71147956/22697a43-0c82-4846-96cd-683ff049bb81)

**Reply by MetzinAround** [reply]
Ah I see. Sorry for the confusion!

**Comment by Mzack9999** [comment]
@darshanpjpt you need to install `lipbcap-dev` dependency which is required internally by `gopacket`

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
